src/messages/layouts/MessageLayoutElement.cpp

```diff
@@ -392,6 +392,7 @@ void ImageWithCircleBackgroundLayoutElement::paint(
     if (pixmap && !this->image_->animated())
     {
         QRectF boxRect(this->getRect());
+        painter.setRenderHint(QPainter::Antialiasing);
```

Comment:
This should be changed to the value it had before, after calling `drawPixmap` (probably best to read it before calling `setRenderHint` rather than pushing the state).

Reply:
I went ahead and added `QPainter::save` and `QPainter::restore` calls around every message layout element paint in [a4daddf](https://github.com/Chatterino/chatterino2/pull/5491/commits/a4daddfd3de952ebfc00b28a9d5b979b6ff15ccc)

Reply:
https://doc.qt.io/qt-6/qpainter.html#save